vernac/declare.ml

```diff
@@ -1068,6 +1068,42 @@ module State = struct
 
   let all pm = ProgMap.bindings pm |> List.map (fun (_,v) -> CEphemeron.get v)
   let find m t = ProgMap.find_opt t m |> Option.map CEphemeron.get
+
+  module View = struct
+    module Obl = struct
+      type t = {
+        name : Id.t
+      ; loc : Loc.t option
```

Comment:
this style is just bad, what do you have against the regular
```suggestion
        name : Id.t;
        loc : Loc.t option;
```
?

Reply:
Style is highly subjective, large industrial OCaml codebases use the "start with ;" style, and it is the one I prefer and the one I'm used to by virtue of my other projects using it.

For me the ; at the beginning is a great visual aid, but again this depends on the person. I'm happy to enforce and style in Coq if you want but that should be done using OCaml Format, as of today both styles are present.

Reply:
I thin the main reason industry uses "start with ;" is that actually otherwise adding ocamldoc comments to record fields just sucks big time, there may be other reasons.

Reply:
That makes sense
This specifc instance bugs me because of empty space before the first feld, maybe we can do 
```suggestion
      { name : Id.t
      ; loc : Loc.t option
```
?

Reply:
Yes, that can indeed be improved, done now.